backend/RoomPlanner/room-planner/src/main/java/edu/roomplanner/rest/UserRestService.java

```diff
@@ -1,33 +1,53 @@
 package edu.roomplanner.rest;
 
-import edu.roomplanner.entity.UserEntity;
+import edu.roomplanner.dto.RoomDto;
 import edu.roomplanner.service.UserService;
+import edu.roomplanner.service.ValidationService;
+import lombok.EqualsAndHashCode;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.List;
 
-@RestController("users")
+@EqualsAndHashCode
+@RestController
 public class UserRestService {
 
+    private static Logger restLogger = LogManager.getLogger(UserRestService.class);
+
     private final UserService userService;
+    private final ValidationService validationService;
 
     @Autowired
-    public UserRestService(UserService userService) {
+    public UserRestService(UserService userService, ValidationService validationService) {
         this.userService = userService;
+        this.validationService = validationService;
     }
 
-    @GetMapping(value = "/users", produces = "application/json")
-    public List<UserEntity> getUsers() {
-        return userService.getUsers();
+    @RequestMapping(method = RequestMethod.GET, value = "/rooms")
+    ResponseEntity<List<RoomDto>> getAllRooms() {
+        restLogger.info("getAllRooms was called.");
+        List<RoomDto> allRoomsList = userService.getAllRooms();
```

Comment:
Remove the 'List' keyword from the name, 'allRooms' is enough, 'List' is redundant